boards/nxp/mr_canhubk3/doc/index.rst

```diff
@@ -296,6 +296,74 @@ For example, to erase and verify flash content:
 
    west flash -r trace32 --startup-args elfFile=build/zephyr/zephyr.elf loadTo=flash eraseFlash=yes verifyFlash=yes
 
+MCUboot
+=======
+
+This board supports app chain-loading using MCUboot.
+
+The command below builds MCUboot and the application together.
+It also generated .hex output files for convenience in flashing.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/flash_shell
+   :board: mr_canhubk3/s32k344/mcuboot
+   :goals: build
+   :build-dir: build
+   :gen-args: --sysbuild
+   :compact:
+
+The resulting artifacts are:
+MCUboot: build/mcuboot/zephyr/zephyr.hex
+App (unsigned): build/flash_shell/zephyr/zephyr.hex
+
+Signing the application
+-----------------------
+
+Preferred (sysbuild)
+Pass your key to sysbuild so the app is signed during the build. For example:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/drivers/flash_shell
+   :board: mr_canhubk3/s32k344/mcuboot
+   :goals: build
+   :build-dir: build
+   :gen-args: --sysbuild -- -DSB_CONFIG_BOOT_SIGNATURE_TYPE_RSA=y -DSB_CONFIG_BOOT_SIGNATURE_KEY_FILE="$PWD/bootloader/mcuboot/root-rsa-2048.pem"
```

Comment:
drop `-- -DSB_CONFIG_BOOT_SIGNATURE_TYPE_RSA=y -DSB_CONFIG_BOOT_SIGNATURE_KEY_FILE="$PWD/bootloader/mcuboot/root-rsa-2048.pem"` these are the defaults